src/Jackett.Common/Indexers/InternetArchive.cs

```diff
@@ -140,9 +143,8 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             var fullSearchUrl = SearchUrl + "?" + qc.GetQueryString();
             var result = await RequestStringWithCookiesAndRetry(fullSearchUrl);
             foreach (var torrent in ParseResponse(result))
-            {
                 releases.Add(MakeRelease(torrent));
```

Comment:
Sorry for revisiting this PR, I just thought of this. We can still merge the PR without this change. But, since the return type is `IEnumerable<ReleaseInfo>` we don't need to cache in a list. 

We could either change this loop to
```
foreach (var torrent in ParseResponse(result))
    yield return MakeRelease(torrent);
```
or remove the loop entirely in favor of a LINQ solution:

`return ParseResponse(result).Select(MakeRelease);`